### PR TITLE
Add `lerna changelog`

### PR DESCRIPTION
**Pull request by hzoo**
From other PR (#23) - Didn't get to updating the readme but I would do that as well

To run it, currently I'm checking out the branch,

`GITHUB_AUTH=public_repo_auth_key_here GITHUB_ORG=babel GITHUB_REPO=babel ./node_modules/.bin/lerna changelog`

---

Example changelog: https://github.com/babel/babel/releases/tag/v6.7.0

Then we go in and manually add code if useful, and summary/explanations

---

I've been using this for the babel changelog

### CODE REVIEW

**Comment by hzoo** [comment]
TODO: figure out what to do with the hardcoded github tags (labels) https://github.com/kittens/lerna/pull/29/files#diff-98b51ca511528ecc68ea54d6adbf8f6bR8

**Comment by hzoo** [comment]
rebased to master

File: lib/commands/changelog/index.js [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+var tags = [
+  "tag: breaking change",
+  "tag: spec compliancy",
+  "tag: new feature",
+  "tag: bug fix",
+  "tag: documentation",
+  "tag: internal",
+  "tag: polish"
+];
```
Comment (gigabo) [inline lib/commands/changelog/index.js]: Whoa, this is cool!  Would definitely like to use this feature.

Comment (hzoo) [inline lib/commands/changelog/index.js]: Yeah, We are using this as a baseline for the babel changelog https://github.com/babel/babel/releases

**Comment by ahfarmer** [comment]
@hzoo have you considered using an existing changelog package like [conventional-changelog](https://github.com/ajoslin/conventional-changelog)?

All you have to do is create the changelog first, then run lerna. One minor change to lerna is required for lerna to pick up the changelog: https://github.com/kittens/lerna/pull/90

**Comment by cdaringe** [comment]
that's pretty cool, @ahfarmer!

**Comment by joscha** [comment]
We currently had to integrate lerna with commitizen and semantic-release.
There is a Commitizen Lerna adapter here: https://github.com/atlassian/cz-lerna-changelog
And a lerna-semantic-release package here: https://github.com/atlassian/lerna-semantic-release
Currently we are not generating a changelog, yet, as semantic-release is tightly coupled to git, but potentially some of the work we did can be reused for this ticket.

**Comment by hzoo** [comment]
Moved to a standalone repo https://github.com/lerna/lerna-changelog so gonna close

**Pull request closed without merging**